src/ascii.rs

```diff
@@ -52,13 +184,13 @@ impl Ascii {
     /// Converts an ascii character into a `u8`.
     #[inline]
     pub fn as_byte(&self) -> u8 {
-        self.chr
+        unsafe{ transmute(*self) }
```

Comment:
This conversion doesn't need an unsafe block because `*self as u8` works as expected.